app/scripts/views/sign_up_password.js

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const AuthErrors = require('lib/auth-errors');
+  const BackMixin = require('views/mixins/back-mixin');
+  const CheckboxMixin = require('views/mixins/checkbox-mixin');
+  const Cocktail = require('cocktail');
+  const CoppaMixin = require('views/mixins/coppa-mixin');
+  const EmailOptInMixin = require('views/mixins/email-opt-in-mixin');
+  const FormPrefillMixin = require('views/mixins/form-prefill-mixin');
+  const FormView = require('views/form');
+  const p = require('lib/promise');
+  const PasswordMixin = require('views/mixins/password-mixin');
+  const ServiceMixin = require('views/mixins/service-mixin');
+  const SignUpMixin = require('views/mixins/signup-mixin');
+  const Template = require('stache!templates/sign_up_password');
+
+  class SignUpPasswordView extends FormView {
+    constructor (options) {
+      super(options);
+
+      this.template = Template;
+      this._account = this.model.get('account');
+    }
+
+    beforeRender () {
+      if (! this._account) {
+        this.navigate('email');
+      }
+    }
+
+    setInitialContext (context) {
+      context.set(this._account.pick('email'));
+    }
+
+    isValidEnd () {
+      return this._doPasswordsMatch();
+    }
+
+    showValidationErrorsEnd () {
+      if (! this._doPasswordsMatch()) {
+        this.displayError(AuthErrors.toError('PASSWORDS_DO_NOT_MATCH'));
```

Comment:
Follow @seanmonstar's lead and [attach the tooltip to the first password field](https://github.com/mozilla/fxa-content-server/pull/5249#discussion-diff-130711885R147).